tpm2/src/util/memory_util.c

```diff
@@ -1,11 +1,11 @@
 /**
- * tpm2_kmyth_misc.c:
+ * memory_util.c:
  *
  * C library containing miscellaneous utilities supporting Kmyth applications
```

Comment:
Part 2 of my comment for the memory utilities header file: In this comment also, "miscellaneous utilities" should probably be changed to something more descriptive, like "memory utilitiies"